jest-puppeteer.config.js

```diff
@@ -0,0 +1,17 @@
+// Ddd this code to improve chrome browser rendering performance when test
```

Comment:
@rinjInTokyo 
typo?

```suggestion
// Add this code to improve chrome browser rendering performance when test
```

Reply:
Sorry, it's a typo